plugin.rb

```diff
@@ -85,20 +85,24 @@ class Engine < ::Rails::Engine
     end
   end
 
-  # DISCOURSE CALENDAR
+  # DISCOURSE CALENDAR HOLIDAYS
 
   add_admin_route 'admin.calendar', 'calendar'
 
   %w[
     ../app/controllers/admin/admin_discourse_calendar_controller.rb
     ../app/controllers/admin/discourse_calendar/admin_holidays_controller.rb
+    ../app/models/discourse_calendar/disabled_holiday.rb
+    ../app/services/discourse_calendar/holiday.rb
   ].each { |path| load File.expand_path(path, __FILE__) }
 
   Discourse::Application.routes.append do
     mount ::DiscourseCalendar::Engine, at: '/'
 
     get '/admin/plugins/calendar' => 'admin/plugins#index', constraints: StaffConstraint.new
     get '/admin/discourse-calendar/holiday-regions/:region_code/holidays' => 'admin/discourse_calendar/admin_holidays#index', constraints: StaffConstraint.new
+    post '/admin/discourse-calendar/holidays/disable' => 'admin/discourse_calendar/admin_holidays#disable', constraints: StaffConstraint.new
+    delete '/admin/discourse-calendar/holidays/enable' => 'admin/discourse_calendar/admin_holidays#enable', constraints: StaffConstraint.new
```

Comment:
It feels a little odd to use `delete` and append `/enable` to the URL. Maybe we can do this instead: 

```suggestion
    delete '/admin/discourse-calendar/holidays/disable' => 'admin/discourse_calendar/admin_holidays#enable', constraints: StaffConstraint.new
```

That also feels a bit odd, to be honest.

Reply:
Not bad! I think that's better than what I had because it's deleting the `disabled_holiday` record. I can make this change! :+1: